gui/WorldView.h

```diff
@@ -52,6 +57,14 @@ class WorldView : public QWidget
   explicit WorldView(QWidget* parent = 0, Qt::WindowFlags flags = 0);
   virtual ~WorldView();
 
+  std::string getActiveDepthMapType();
+
+  void setActiveDepthMap(vtkMaptkCamera *cam, QString vtiPath);
+  void enableDepthMap();
+
+signals:
+  void updateThresholds(double,double,double,double);
```

Comment:
This is not an appropriate name for a signal; it is imperative tense. Signals shall be past tense. An appropriate name is e.g. `thresholdsChanged`. (`depthThresholdsChanged` is probably better.)

(Note: *slots* shall be imperative tense.)